src/main/java/io/lettuce/core/LcsArgs.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2011-Present, Redis Ltd. and Contributors
+ * All rights reserved.
+ *
+ * Licensed under the MIT License.
+ *
+ * This file contains contributions from third-party contributors
```

Comment:
FWIW You only need the first 5 lines of the license agreement as this is a new file